### PR TITLE
docs(tem): tem x serverless tutorial

### PR DESCRIPTION
**Pull request by nerda-codes**

### CODE REVIEW

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+## Clone the GitHub repository
+
+We have created a [GitHub repository](https://github.com/quantumsheep/scw-demo-tem) that contains all the code you need to deploy your infrastructure. Open a terminal and clone the repository to your local machine using the following command:
+
+```shell
+git clone https://github.com/quantumsheep/scw-demo-tem
+```
+
+### Files and directories
+
```
Comment (RoRoJ) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:36]: I think the tuto is missing numbered steps?

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- Finding latest version of scaleway/scaleway...
+- Installing hashicorp/local v2.4.0...
+- Installed hashicorp/local v2.4.0 (signed by HashiCorp)
+- Installing hashicorp/archive v2.3.0...
+- Installed hashicorp/archive v2.3.0 (signed by HashiCorp)
+- Installing scaleway/scaleway v2.16.3...
+- Installed scaleway/scaleway v2.16.3 (signed by a HashiCorp partner, key ID F5BF26CADF6F9614)
+
+Partner and community providers are signed by their developers.
+If you'd like to know more about provider signing, you can read about it here:
```
Comment (RoRoJ) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:77]:
```suggestion
<Message type="tip">
If you'd like to know more about provider signing, you can read about it in the [official Terraform documentation](https://www.terraform.io/docs/cli/plugins/signing.html).
</Message>
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+https://www.terraform.io/docs/cli/plugins/signing.html
```
Comment (RoRoJ) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:78]:
```suggestion
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+Terraform has created a lock file .terraform.lock.hcl to record the provider
+selections it made above. Include this file in your version control repository
+so that Terraform can guarantee to make the same selections by default when
+you run "terraform init" in the future.
```
Comment (RoRoJ) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:83]:
```suggestion
you run `terraform init` in the future.
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
```
Comment (RoRoJ) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:87]:
```suggestion
You may now begin working with Terraform. Try running `terraform plan` to see
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx (hunk 4 above)
Comment (nedaehm) [reply tutorials/send-emails-with-serverless-and-tem/index.mdx:83]: That's part of the output

File: tutorials/send-emails-with-serverless-and-tem/index.mdx (hunk 5 above)
Comment (nedaehm) [reply tutorials/send-emails-with-serverless-and-tem/index.mdx:87]: This is part of the output, not sure I can edit it

File: tutorials/send-emails-with-serverless-and-tem/index.mdx (hunk 2 above)
Comment (nedaehm) [reply tutorials/send-emails-with-serverless-and-tem/index.mdx:77]: This is part of the output, not sure it'll be clear if I add a message box

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+  - You have a domain name
+  - You have [downloaded and installed Terraform](https://developer.hashicorp.com/terraform/downloads)
+</Message>
+
+## Clone the GitHub repository
+
+We have created a [GitHub repository](https://github.com/quantumsheep/scw-demo-tem) that contains all the code you need to deploy your infrastructure. Open a terminal and clone the repository to your local machine using the following command:
+
+```shell
+git clone https://github.com/quantumsheep/scw-demo-tem
```
Comment (quantumsheep) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:32]:
```suggestion
git clone https://github.com/scaleway/tutorial-tem-serverless-terraform
```
I ported the repository into our official org

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+  validation: 2023-04-17
+  posted: 2023-05-01
+---
+
+In this tutorial, you will learn how to send emails with [Transactional Email](/managed-services/transactional-email/quickstart/) and [Serverless Functions](/serverless/functions/quickstart/) using a Terraform configuration file that builds your entire infrastructure such as a Serverless Function and a website with a registration page hosted on an Object Storage bucket. It also allows you to configure SPF, DKIM and MX records on your sending domain.
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console.](https://console.scaleway.com)
```
Comment (quantumsheep) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:22]:
```suggestion
  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+  - You have a domain name
+  - You have [downloaded and installed Terraform](https://developer.hashicorp.com/terraform/downloads)
```
Comment (quantumsheep) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:24]:
```suggestion
  - You have [installed Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+</Message>
+
+## Clone the GitHub repository
+
+We have created a [GitHub repository](https://github.com/quantumsheep/scw-demo-tem) that contains all the code you need to deploy your infrastructure. Open a terminal and clone the repository to your local machine using the following command:
```
Comment (quantumsheep) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:29]:
```suggestion
We have created a [GitHub repository](https://github.com/scaleway/tutorial-tem-serverless-terraform) that contains all the code you need to deploy your infrastructure. Open a terminal and clone the repository to your local machine using the following command:
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+
+```shell
+git clone https://github.com/quantumsheep/scw-demo-tem
+```
+
+### Files and directories
+
+In the `scw-tem-demo` directory, you will find three folders: `infra`, `signup-handler` and `super-product`.
```
Comment (quantumsheep) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:37]:
```suggestion
In the `tutorial-tem-serverless-terraform` directory, you will find three folders: `infra`, `signup-handler` and `super-product`.
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+- The `super-product` folder holds an `index.html.tftpl` file which is a Terraform template. This file contains an HTML template for the registration page of the website we are creating. This template allows you to automatically create and configure the SPF, MX and DKIM records needed for your domain, as well as a Serverless Function.
+
+<Message type="tip">
+  If you want to use a specific function, you can replace `${DISPATCHER_ENDPOINT}` (line 74) in the `index.html.tftpl` file with your function's endpoint.
+</Message>
+
+
+## Build and deploy your infrastructure
+
+Open a terminal and go to the `scw-tem-demo` directory using the following command:
```
Comment (quantumsheep) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:50]:
```suggestion
Open a terminal and go to the `tutorial-tem-serverless-terraform` directory using the following command:
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+
+```shell
+cd scw-tem-demo/infra
```
Comment (quantumsheep) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:53]:
```suggestion
cd tutorial-tem-serverless-terraform/infra
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+
+If you ever set or change modules or backend configuration for Terraform,
+rerun this command to reinitialize your working directory. If you forget, other
+commands will detect it and remind you to do so if necessary.
+```
+
+Run the following command to deploy your infrastructure:
+
+```shell
+
```
Comment (quantumsheep) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:99]:
```suggestion
```

File: tutorials/send-emails-with-serverless-and-tem/index.mdx [hunk 14]
```diff
@@ -0,0 +1,155 @@
...
+Log in to your email address. Your email titled **Welcome to my cool product!** displays in your mailbox.
+<Lightbox src="scaleway-tem-email.webp" alt=""/>
+
+
+
+
+
+
+
+
```
Comment (quantumsheep) [inline tutorials/send-emails-with-serverless-and-tem/index.mdx:155]:
```suggestion
```

**Review by quantumsheep — CHANGES_REQUESTED** [review]

**Pull request merged**